test/functional/shada/merging_spec.lua

```diff
@@ -13,6 +13,12 @@ local read_shada_file = shada_helpers.read_shada_file
 local wshada, sdrcmd, shada_fname =
   get_shada_rw('Xtest-functional-shada-merging.shada')
 
+local mock_file_path = '/a/b/'
+local header = '\xc4\x06'
```

Comment:
AFAIR `\xXX` is supposed to be unavailable in some circumstances. And in any case this is inconsistent with the rest of the file.

Reply:
Thanks @ZyX-I . @aignas FYI I reverted this in the merge.